test/test_device_group_names.py

```diff
@@ -20,8 +20,16 @@ def test_authentication(self, helper):
         c_terminator.group = ""
         recording = clare.uploads_recording_for(c_terminator)
 
+        with pytest.raises(UnprocessableError):
+            c_terminator = helper.given_new_device(
+                None,
+                "1234",
+                clares_group,
+                description="and devicename 1234 fails, for it must containt alpha char",
+            )
+
     # use this when we make group + device unique
-    # def test_authentication(self, helper):
+    # def test_unique_authentication(self, helper):
```

Comment:
Rather than comment out the tests, use pytest's helpers for skipping tests. These take a reason which will be displayed when the test is skipped.

https://docs.pytest.org/en/latest/skipping.html

This will make it clearer that there are some tests that are intentionally being skipped (makes them harder to forget).